federator/src/lib/Federator.js

```diff
@@ -1,274 +1,365 @@
-const web3 = require('web3');
-const fs = require('fs');
-const TransactionSender = require('./TransactionSender');
-const CustomError = require('./CustomError');
-const BridgeFactory = require('../contracts/BridgeFactory');
-const FederationFactory = require('../contracts/FederationFactory');
-const AllowTokensFactory = require('../contracts/AllowTokensFactory');
-const utils = require('./utils');
+const web3 = require("web3");
+const fs = require("fs");
+const TransactionSender = require("./TransactionSender");
+const CustomError = require("./CustomError");
+const BridgeFactory = require("../contracts/BridgeFactory");
+const FederationFactory = require("../contracts/FederationFactory");
+const AllowTokensFactory = require("../contracts/AllowTokensFactory");
+const utils = require("./utils");
+const utilsTs = require("./utilsTs");
```

Comment:
maybe `typescriptUtils` so it's clearer? Idk maybe it's just me

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4